install/helm/agones/templates/crds/_gameserverstatus.yaml

```diff
@@ -65,6 +65,40 @@ status:
           nullable: true
           items:
             type: string
+{{- if .featureCountsAndLists }}
+    counts:
```

Comment:
maybe more of a design question, but this whole feature seems like it belongs in .Status rather then .Spec as it reflects the ongoing gameserver status. Here it's reflected in both?

Reply:
Oh, I see your point. Can a counter/list only be started in the Spec, or will we allow new counters/lists at any given time?

Reply:
Ideally allow new counters/lists at any given time? Be able to start up a gameserver with a counter/list, or add one on later.

Reply:
There's also the overall size limits for the CRD to consider too. Letting users have 1000 lists of 1000 values each in multiple places will leave the potential for users to go over the CRD size limits.

Reply:
From the design: 

> Out of scope: The ability to add/edit or delete GameServer keys for counters and list at runtime. 

So new counters/lists at any given time seems not allowed.

Reply:
I think it makes sense to have this in both Spec and Status. In Spec it's the initial value while in Status it's the current value. The property name and comment are a little confusing though, i.e. in line 167 of `_gameserverspecschema.yaml`, I think the comment should be `initial count` rather than `current count`.